src/Sylius/Bundle/ApiBundle/DataProvider/AddressItemDataProvider.php

```diff
@@ -54,7 +54,7 @@ public function getItem(string $resourceClass, $id, string $operationName = null
             in_array('ROLE_USER', $user->getRoles(), true) &&
```

Comment:
It is not part of this PR but maybe you could change something so `$user instanceof ShopUserInterface` won't be called 2 times? 🚀

Reply:
maybe :D but like you said it isn't the scope of this PR :)